docs/developer_guide/setup.md

```diff
@@ -17,6 +17,7 @@ MedModels utilizes a `Makefile` to manage development tasks. Here's a breakdown
 - **install-docs:** Similar to install, but additionally installs development dependencies needed for building the docs.
 - **build-dev:** Builds the Rust crate and installs it as a python module (using `maturin develop`).
 - **test:** Runs both Python (using `pytest`) and Rust unit tests (using `cargo test`).
+- **test-python-coverage:** Runs Python tests with detailed output and generates a line-by-line coverage report.
```

Comment:
Remove the detailed output part. Both testing functions provide "detailed" output, but this is not an important information anyway imo.
I also don't think a line-by-line coverage report is accurate. it just shows the
> [...] line numbers of statements in each module that weren't executed.